runtime/protocol/http-client/api/http-client.api

```diff
@@ -142,6 +142,9 @@ public final class aws/smithy/kotlin/runtime/http/middleware/HttpResponseExcepti
 	public final fun setStatusCode (Laws/smithy/kotlin/runtime/http/HttpStatusCode;)V
 }
 
+public final class aws/smithy/kotlin/runtime/http/middleware/ResolveEndpointKt {
+}
+
```

Comment:
**Question**: Why is this here?

Reply:
It appeared to be a missed api snapshot update, i'm not sure why things downstream were building without it.

Reply:
That doesn't seem right. There's no **ResolveEndpoint.kt** file in [**aws/smithy/kotlin/runtime/http/middleware**](https://github.com/awslabs/smithy-kotlin/tree/main/runtime/protocol/http-client/common/src/aws/smithy/kotlin/runtime/http/middleware) and this PR doesn't add one. CI is failing too. Is your local copy combining two changes?

Reply:
Yeah you're right. Looks like it was a build artifact in my working copy. Cleaned, re-built, it cleared out of the snapshot -- pushed.